platform/darwin/src/MGLGeometry.h

```diff
@@ -14,6 +14,11 @@ typedef struct __attribute__((objc_boxable)) MGLCoordinateSpan {
     CLLocationDegrees longitudeDelta;
 } MGLCoordinateSpan;
 
+/* Defines a point on the map in Mercator projection for a specific zoom level. */
+typedef struct __attribute__((objc_boxable)) MGLMapPoint {
```

Comment:
Fwiw, we’ve so far only marked structs boxable after adding a category method and property to NSValue (see NSValue+MGLAdditions.h). I guess it isn’t a big deal to mark this type as being boxable in an NSValue, but it is a good practice to provide NSValue conversions for types we expect to put in an NSValue.